gulpfiles/prod.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var gulp = require('gulp');
+var fileinclude = require('gulp-file-include');
+var marked = require('gulp-marked');
+
+var buildDirectory = '../build/';
+
+gulp.task('build', ['build:homepage']);
+
+gulp.task('build:homepage', ['build:readme-html'], function() {
+    var wwwBuildDirectory = buildDirectory + 'www';
```

Comment:
Just my style. I'm allergic to magic constants. I try to give everything an identifying name so that comments are less necessary over time. But I know JavaScripters are more flexible about that, so I'll change it.